misc/services/mobile-webui/mobile-webui-frontend/src/components/ScanHUAndGetQtyComponent.jsx

```diff
@@ -217,6 +217,7 @@ const ScanHUAndGetQtyComponent = ({
     bestBeforeDate,
     lotNo,
     productNo,
+    ean13ProductCode,
```

Comment:
hmmm, from where this is coming? in ean13.js/parseEAN13CodeString I see no ean13ProductCode.
i think it's not needed